source/common/types/logging.types.js

```diff
@@ -48,3 +50,44 @@ export type LogSystemInfoParams = {
   ram: string,
   startTime: string,
 };
+
+export type LogStateSnapshotParams = {
```

Comment:
I can see most of the props are the same as the Diagnostics component. For clarity, you could do the following:

At `components/status/DaedalusDiagnostics.js`:

```diff
- type Props = {
+ export type DaedalusDiagnosticsProps {
```

Then here:

```js
export type LogStateSnapshotParams = {
  ...$Exact<DaedalusDiagnosticsProps>,
 // add the remaining props
}
```

Reply:
Done

Reply:
This was not a good idea as the props are not equal.
Please revert this @DeeJayElly 
cc @daniloprates